src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The stylesheet link references a `.scss` file, which is not directly usable in HTML. You should compile the SCSS file to CSS and link the resulting `.css` file instead.